workflow_avamb/src/rip_bins.py

```diff
@@ -268,8 +268,6 @@ def make_all_components_pair(
                         
 
 
-            #dsc_sorted_weights: list[float] = [min(weight_edges.keys())]
-
```

Comment:
There are a whole bunch of `print` statements in this script, I don't understand why. Is it for logging? Debugging? If so, shouldn't it print to a log file instead of stdout? It's not wrong, I just don't get it.